### ISSUE TITLE
Not working with `forwardRef` wrapped components

### ISSUE THREAD

**Issue by leepowellcouk** [storybooks/babel-plugin-react-docgen#67]
https://github.com/storybooks/babel-plugin-react-docgen/blob/fa170793df1d9065ba4bd25238627a72e55fbcb6/src/index.js#L49

This seems to exit early when a component is generated in a `forwardRef` call:

```
const Component = React.forwardRef(() => {
  return <div>Hello</div>
});
```

And as such does not attach any docgen data. This seems to stem from `actualNameHandler`.

**Comment by leepowellcouk** [comment]
Looks like v4 of react-docgen support this: https://github.com/reactjs/react-docgen/pull/324

**Comment by leepowellcouk** [comment]
Just tested this plugin using `react-docgen@4` and can confirm a component wrapped in forwardRef works as expected if this is commented out:
```
if (!exportName) {
  // return;
}
```
So if we can fix `actualNameHandler` to recognise forwardRefs this should work.

**Comment by shilman** [comment]
any chance you can issue a PR for this? If so I can test and release an upgrade

**Comment by leepowellcouk** [comment]
Yes, I'm just working one up, will be with you shortly.

**Comment by leepowellcouk** [comment]
Added: https://github.com/storybooks/babel-plugin-react-docgen/pull/68

**Comment by danielduan** [comment]
fixed in `v3.0.0`, will integrate into storybook.

**Comment by leepowellcouk** [comment]
Do you want me to create a PR to update to v3 here:
https://github.com/storybooks/storybook/blob/next/app/react/package.json
